doc/src/vpr/fasm.rst

```diff
@@ -0,0 +1,264 @@
+FPGA Assembly (FASM) Output Support
+===================================
+
+VPR can emit a FASM_ file to represent
+the place and routed design via FASM metadata encoded in the VPR architecture
+definition and routing graph.  The output FASM file can then be converted into
+the target architecture via architecture specific tooling.
+
+.. _FASM: https://github.com/SymbiFlow/fasm
+
+VPR metadata
+------------
+
+VPR allows for ``metadata`` blocks
```

Comment:
Similarly, probably want to rephrase this, something like: 'genfasm uses metadata blocks ....'